src/main/java/de/tum/in/www1/artemis/service/compass/controller/AutomaticAssessmentController.java

```diff
@@ -57,6 +58,7 @@ public void addFeedbacksToAssessment(AssessmentIndex index, Map<String, Feedback
      * @param modelIndex      manages all models
      * @param assessmentIndex manages all assessments
      */
+    // TODO CZ: only assess models automatically that do not already have a complete manual assessment?
```

Comment:
Is this TODO already solved?

Reply:
No, it is not. We would have to query the database here to check if the models already have a manual result or not, but I did not want to add an additional dependency to the repository here. Also I am not really sure, if we even save much time then. 
I leave the TODO for now as I would like to further investigate and discuss this.